templates/error/error403.html.twig

```diff
@@ -0,0 +1,12 @@
+{% block stylesheets %}
+    {{ encore_entry_link_tags('error') }}
+{% endblock %}
+
+<div class="p-5">
+	<div class="p-lg-5 p-0">
+		<h1 class="h1 text-danger">Erreur 403</h1>
+		<h2 class="h2 mb-5">Vous n'avez pas les droits d'accès pour cette page</h2>
+
+		<a class="btn btn-lg btn-primary" href="{{ path('home') }}">Retourner à l'accueil</a>
+	</div>
+</div>
```

Comment:
this could extend your layout so that you coud still have the navbar and navigate the site when you get a 403 error response